useful-scripts/get_data_block_responses/get_data_block_responses.py

```diff
@@ -183,7 +184,7 @@ def write_this_block(status_code, response_time, response_dict):
     write_this_block(
         status_code=resp.status_code,
         response_time=block_time_end - block_time_start,
-        response_dict=json_response)
+        response_dict=json_response.pop('location', None))
```

Comment:
Maybe just double check this as I tried this in a test script and it didn't work

```
response = {
  "results": [
    {
      "id": "1",
      "locationId": "L1",
      "location": {
        "country": {
          "code": "1",
          "name": "England"
        }
      }
    },
    {
      "id": "2",
      "locationId": "L2",
      "location": {
        "country": {
          "code": "2",
          "name": "Wales"
        }
      }
    }
  ]
}

response.pop("location", None)

print(response)
```